extensions/amp-access/amp-access-spec.md

```diff
@@ -18,9 +18,43 @@ limitations under the License.
 
 ### <a name="amp-access-spec"></a>AMP Access Spec
 
-#Background
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>AMP Access or “AMP paywall and subscription support” gives publishers control over which content can be accessed by a reader and with what restrictions, based on the reader’s subscription status, number of views, and other factors.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><a href="/docs/reference/experimental.html">Experimental</a></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-access" src="https://cdn.ampproject.org/v0/amp-access-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://github.com/ampproject/amphtml/blob/master/examples/article-access.amp.html">article-access.amp.html</a></td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-access-spec` tag
+(see also `amp-access-spec` in the [AMP validator specification](https://github.com/ampproject/amphtml/blob/master/validator/validator.protoascii):
 
-AMP Access or “AMP paywall and subscription support” provides control to publishers over what content can be accessed by a reader and with what restrictions,  based on the reader’s subscription status, number of views and so on.
+<!---
+More validations need to be added once implemented;
+this table will change soon.
+-->
+
+<table>
+  <tr>
+    <th width="40%"><strong>Validation Error</strong></th>
+    <th>Description</th>
```

Comment:
Another important one is MANDATORY_ATTR_MISSING or INVALID_ATTR_VALUE for the attribute `type="application/json"`